docs/running-notebooks/opening-sessions/index.md

```diff
@@ -25,6 +25,19 @@ https://reana.cern.ch/f8be55e4-5d18-43f9-b977-f773fdcab163?token=<your-reana-tok
 It could take several minutes to start the interactive session.
 ```
 
+!!! warning
+    In order for the custom notebook image you supply to seamlessly integrate
+    with REANA, it must:
+
+    - have Jupyter installed
+    - expose port `8888`
+    - use the `HTTPS` protocol
+    - provide the [`start-notebook.sh`](https://github.com/jupyter/docker-stacks/blob/main/base-notebook/start-notebook.sh) script
```

Comment:
I'm not sure whether it is better to mention this script and then recommend below to use an image based on one of the official Jupyter images, or to directly say that, as a requisite, the custom image should be based upon an official Jupyter one. What do you think?